packages/default-storage-expo-plugin/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@react-native-async-storage/expo-plugin",
+  "version": "1.0.0",
+  "description": "Asynchronous, persistent, key-value storage system for React Native.",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "files": [
+    "build/",
+    "src/",
+    "app.plugin.js"
+  ],
+  "author": "Two Doors Dev <hello@twodoors.dev>",
+  "contributors": [
+    "Hassan Khan <h@twodoors.dev> (https://github.com/hassankhan)",
+    "Bill Spooner <b@twodoors.dev> (https://github.com/sandyklark)"
+  ],
+  "homepage": "https://github.com/react-native-async-storage/async-storage#readme",
+  "license": "MIT",
+  "keywords": [
+    "react-native",
+    "react native",
+    "async storage",
+    "asyncstorage",
+    "storage",
+    "expo",
+    "expo config plugin"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/react-native-async-storage/async-storage.git",
+    "directory": "packages/default-storage-expo-plugin"
+  },
+  "scripts": {
+    "prepack": "yarn build",
+    "prepare": "expo-module prepare",
+    "prepublishOnly": "expo-module prepublishOnly",
+    "build": "CI=true expo-module build",
```

Comment:
why set `CI` env here?

Reply:
Unfortunately `expo-module build` always starts `tsc` in watch mode, and this was to get around that. It looks like `expo-module build` will [skip watch mode when either `CI` or `EXPO_NONINTERACTIVE` is set](https://github.com/expo/expo/blob/main/packages/expo-module-scripts/bin/expo-module-build#L30C33-L32); I can switch it to `EXPO_NONINTERACTIVE` instead?